### PR TITLE
pivot: get group element sizes directly

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: js/ui/pivot_grid/ui.pivot_grid.area_item.js [hunk 1]
```diff
@@ -402,23 +402,29 @@ export const AreaItem = Class.inherit({
...
+
+    getGroupWidth: function() {
+        return this._getGroupElementSize('width');
+    },
+
+    _getGroupElementSize(dimension) {
+        const size = this.groupElement()[0].style[dimension];
+
+        if(size.indexOf('px') >= 0) {
+            return Number(size.replace('px', ''));
```
Comment (IgnatovDan) [inline js/ui/pivot_grid/ui.pivot_grid.area_item.js]: TODO: parseHeight/normalizeStyleProp/parsePixelValue ?

File: js/ui/pivot_grid/ui.pivot_grid.area_item.js [hunk 2]
```diff
@@ -402,23 +402,29 @@ export const AreaItem = Class.inherit({
     },
 
     setGroupWidth: function(value) {
-        if(value >= 0) {
-            this._groupWidth = value;
-            this._groupElement[0].style.width = value + 'px';
-        } else {
-            this._groupElement[0].style.width = value;
-        }
+        this.groupElement()[0].style.width = isNumeric(value) ? `${value}px` : value;
```
Comment (IgnatovDan) [inline js/ui/pivot_grid/ui.pivot_grid.area_item.js]: TODO: setHeight/normalizeStyleProp ?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/ui/pivot_grid/ui.pivot_grid.area_item.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/ui/pivot_grid/ui.pivot_grid.area_item.js]: none of these methods satisfy the current implementation

File: js/ui/pivot_grid/ui.pivot_grid.area_item.js (hunk 2 above)
Comment (EugeniyKiyashko) [reply js/ui/pivot_grid/ui.pivot_grid.area_item.js]: fixed

**Review by IgnatovDan — DISMISSED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**